cliquet/cache/postgresql/__init__.py

```diff
@@ -53,7 +59,7 @@ class Cache(CacheBase):
         of connections used in a multi-process deployment.
 
     :noindex:
-    """
+    """  # NOQA
```

Comment:
Because of the 80 chars limit of the default ``cliquet.cache_poolclass`` line :]